vtex-apps/login
# Feature/ch30673/redirect prop

#### What is the purpose of this pull request?

Add prop to redirect to "/login" instead of rendering login popover

#### What problem is this solving?

This PR:
- Changes the log in to be a button instead of a link in mobile (or when the new prop is used)
- Adds the `logInButtonBehavior` prop which defaults to `popover` and can be changed to `link`
- Adds the new `logInButtonBehavior` prop to the site-editor
- Fixes the translation of the `mirrorTooltipToRight` prop in the site-editor

#### How should this be manually tested?

Go to
https://rafaprtest3--storecomponents.myvtex.com/admin/cms/site-editor
and choose `Login` in the right sidebar. In this page, you can see the `Mirror tooltip to right` toggle correctly translated. Then change the `Log in button behavior` select to the value `link`. If you try clicking the `Sign in` button in the store header, it should navigate you to the page `/login` instead of opening a popover.

#### Screenshots or example usage

After clicking the `Sign in` button:

![image](https://user-images.githubusercontent.com/22064061/74267916-b1a7b000-4ce5-11ea-951e-4015e99f82ca.png)

![image](https://user-images.githubusercontent.com/22064061/74267927-b79d9100-4ce5-11ea-9799-23e1b1abf942.png)

#### Types of changes
